sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/event_processor.py

```diff
@@ -239,7 +244,7 @@ def _load_balancing(self):
                     self._consumer_group,
                     self._load_balancing_interval,
                 )
```

Comment:
We're already logging a warning in `def _process_error` so this will log double warnings. Was that intended?

Reply:
you're right, we should improve it so just log once.